cli/wallet/wallet.go

```diff
@@ -118,6 +119,10 @@ func NewCommands() []cli.Command {
 				Flags: []cli.Flag{
 					walletPathFlag,
 					decryptFlag,
+					cli.StringFlag{
+						Name:  "address",
```

Comment:
I'd interpret this parameter as "dump data for this address only", which actually is a useful thing to have, but it's a different functionality.

This option should be something as `extract-keys, k` or (if we don't use JSON output format that is expected from the `dump`) a separate command.

Reply:
I think `dump-keys [-a ..]` is ok.

Reply:
Yep, except `-a` is not supported at the moment.